Update docs about grouping PRs in next version (#6970)
elastic/integrations: docs/developer_workflow_design_build_test_integration.md

It is up to you, as the package developer, to decide how many changes you want to release in a single version.
For example, you could implement a change in a PR and bump up the package version in the same PR. Or you could
implement several changes across multiple PRs and then bump up the package version in the last of these PRs

> or in a separate follow up PR. As an example, it could be followed this procedure for a package that its latest published version is `2.5.0`:

Reviewer: Nit
```suggestion
or in a separate follow up PR. For example, you can apply the following procedure for a package whose latest published version is `2.5.0`:
```
